src/Microsoft.AspNetCore.Razor.Design/buildCrossTargeting/Microsoft.AspNetCore.Razor.Design.targets

```diff
@@ -0,0 +1,20 @@
+﻿<!--
+Copyright (c) .NET Foundation. All rights reserved.
+Licensed under the Apache License, Version 2.0.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+**********************************************************************************
+Target: ResolveRazorTargetFramework
+
+Outputs the Razor target framework used by tooling to the console.
+
+**********************************************************************************
+-->
+  <Target Name="ResolveRazorTargetFramework">
+    <MSBuild Projects="$(MSBuildProjectFile)"
+             Targets="ResolveRazorTargetFramework"
+             Properties="TargetFramework=$(TargetFrameworks.Split(';')[0])">
```

Comment:
If `TargetFrameworks` is unset and `TargetFramework` is set the other `build/Microsoft.AspNetCore.Razor.Design.targets` will be called. This bit normalizes those call paths. CSproj is either cross framework targeting or not.